src/lib/settings/Gateway.js

```diff
@@ -5,30 +5,31 @@ const SchemaFolder = require('./SchemaFolder');
 const discord = require('discord.js');
 
 /**
+ * <info>You should never create a Gateway instance by yourself, as it manages many data internally.
```

Comment:
danger, "You should never create a Gateway instance directly. Please check {@link UnderstandingSettingGateway} to learn how to add your own Gateway."